pyproject.toml

```diff
@@ -59,7 +59,7 @@ colorlog = { version = ">=5.0.1,<7.0.0", optional = true }
 colorama = { version = "^0.4.4", optional = true }
 importlib-metadata = { version = "<4.8.1", python = "<3.8" }
 jinja2-ansible-filters = "^1.3.2"
-ruamel-yaml = "^0.17.32"
+ruamel-yaml = ">=0.17.32,<0.19.0"
```

Comment:
Huh, why is dependabot adding an upper bound to the version here? I never saw it to do that!

Cc @enkeys @michalxo